integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -298,6 +298,117 @@ def get_documents_by_id(
             documents.append(self.qdrant_to_haystack.point_to_document(record))
         return documents
 
+    # def query_hybrid(
```

Comment:
Commented because not really working. Search_Batch return ScoredPoint and not Point, so they are not convertible to Haystack documents (no payload)